phoenix-core/src/it/java/org/apache/phoenix/end2end/index/GlobalIndexCheckerIT.java

```diff
@@ -95,7 +97,7 @@ public void beforeTest(){
         indexOptionBuilder = new StringBuilder();
         if (!encoded) {
             optionBuilder.append(" COLUMN_ENCODED_BYTES=0");
-            indexOptionBuilder.append(" IMMUTABLE_STORAGE_SCHEME=SINGLE_CELL_ARRAY_WITH_OFFSETS, COLUMN_ENCODED_BYTES=2");
+            //indexOptionBuilder.append(" IMMUTABLE_STORAGE_SCHEME=SINGLE_CELL_ARRAY_WITH_OFFSETS, COLUMN_ENCODED_BYTES=2");
```

Comment:
On purpose?

Reply:
No, it was on purpose. I wanted to make sure that it works without the single cell format and forgot to remove it.  I will remove it.